blog-cse/2024-10-04-content.md

```diff
@@ -0,0 +1,221 @@
+---
+title: October 10, 2024 - Content Release
+hide_table_of_contents: true
+keywords:
+  - log mappers
+  - log parsers
+  - detection rules
+image: https://help.sumologic.com/img/sumo-square.png  
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-cse/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+This content release includes:
+* Detection rules centered around Amazon Bedrock activities
+* Consolidation of AWS CloudTrail mappers to replicate current mapper behavior with fewer distinct mappers.
+* New support for GitHub Enterprise Audit (parsing and mapping)
+* New support for Honeywell Pro-Watch (parsing and mapping)
+* New support for Citrix Zendesk (parsing and mapping)
+* Further mapping updates to better employ [Normalized Classification](https://help.sumologic.com/docs/cse/schema/cse-normalized-classification/) fields across data sources
+* Removal of some duplicate mapped fields
+* Other changes enumerated below
+
+## Rules
+- [New] MATCH-S00922 AWS Bedrock Agent Created
+    - This rule detects when an AWS Bedrock Agent has been created in the environment. Bedrock Agents can be configured with various parameters to build AI applications. Take a look at the "responseElements.agent.agentName" field to see the name of the agent being created. Ensure that the user creating the agent is authorized to develop AI applications within the environment.
+- [New] MATCH-S00924 AWS Bedrock Guardrail Deleted
+    - AWS Bedrock Guardrails provide users with the ability to configure options like filtering out harmful content or defining denied topics for models. Guardrails also allow the blocking of sensitive information such as PII. Ensure that this deletion was performed by an authorized user during an expected change. Look at other activity from this user account, focusing on the Bedrock service and pivoting from there if the event is deemed suspicious.
+- [New] MATCH-S00923 AWS Bedrock Model Invocation Denied for User
+    - A user has attempted to invoke a model via AWS Bedrock for which access was denied due to a permission issue. This event can be a normal occurrence for a user who has not been provisioned the proper IAM resources for AWS Bedrock. However, it could also be a malicious attempt at running a particular model via AWS Bedrock. Take a look at the username, IP address, role type, role and model via the fields["requestParameters.modelId"] field.
```

Comment:
```suggestion
    - A user has attempted to invoke a model via AWS Bedrock for which access was denied due to a permission issue. This event can be a normal occurrence for a user who has not been provisioned the proper IAM resources for AWS Bedrock. However, it could also be a malicious attempt at running a particular model via AWS Bedrock. Take a look at the username, IP address, role type, role and model via the "requestParameters.modelId" field.
```